docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.8'
```

Comment:
double check, by searching docker docs, but I guess "version" parameter is obsolete.